### ISSUE TITLE
life_history from ABRS_2022 for Pimeleas

### ISSUE THREAD

**Issue by wcornwell** [traitecoevo/austraits.build#669]
Seems like ABRS_2022 life history might have an issue somewhere
---

Brief description of the problem

```r
austraits$traits %>%
  filter(grepl("Pimelea brevifolia",taxon_name)) %>%
  filter(trait_name == "life_history") %>%
  select(dataset_id,taxon_name,trait_name,value)
```

I bunch of Pimeleas are coming in as annuals, which I think are actually shrubs/subshrubs?
<img width="596" alt="Screenshot 2023-02-20 at 3 00 55 pm" src="https://user-images.githubusercontent.com/5534161/220006517-d263f442-aa32-44c6-b076-cc07d1abd15e.png">
<img width="561" alt="Screenshot 2023-02-20 at 2 59 24 pm" src="https://user-images.githubusercontent.com/5534161/220006524-b1c948bf-581b-46dc-ac28-36269ca6f072.png">

**Comment by wcornwell** [comment]
this problem seems to be confined to rice flowers, strangely

**Comment by ehwenk** [comment]
Yes, we're aware of this. I've already added it to the "to fix" list. It has to do with a wording in the Flora of Australia (and PlantNet) that we didn't catch.

**Comment by dcol2804** [comment]
This should be fixed now with AusTraits release 4.2.0. Phew!
